client/src/modules/employees/registry/search.modal.html

```diff
@@ -0,0 +1,184 @@
+<form
+  name="ModalForm"
+  ng-submit="ModalCtrl.submit(ModalForm)"
+  data-modal="employee-search"
+  bh-form-defaults
+  novalidate>
+
+  <div class="modal-header" style="box-shadow : 0px 2px 4px -4px black; z-index:1000">
```

Comment:
Normally when CSS is used in multiple locations, it is good practice to make it into a CSS `class`.  That way this element, and future elements, can benefit from sharing the same code base, the same bug fixes, and the whole application becomes smaller.

This is a non-blocking comment.